Vagrantfile

```diff
@@ -41,6 +45,12 @@ Vagrant.configure("2") do |config|
         raise Vagrant::Errors::VagrantError.new, "Incorrect number of edge cluster coordinates."
     end
 
+    # Fetches CoreDNS remote repo and moved to correct branch.
+    if !File.directory?($coredns_path)
+        system("git clone https://github.com/optikon/coredns " + $coredns_path)
+    end
+    system("cd " + $coredns_path + " && git checkout " + $coredns_branch)
```

Comment:
We should not always switch the branch, that information should only be used if we have to pull the code down.